packages/dm-core-plugins/src/generic-list/GenericTablePlugin.tsx

```diff
@@ -47,27 +62,26 @@ export const GenericTablePlugin = (
   }, [document, loading])
 
   const deleteItem = (reference: string, key: string) => {
-    dmssAPI.documentRemove({ idReference: reference }).then(() => {
-      delete items[key]
-      setItems({ ...items })
-    })
+    delete items[key]
+    setItems({ ...items })
+    setDirtyState(true)
   }
-  const addItem = (reference: string, type: string) => {
+  const addItem = (reference: string) => {
     dmssAPI
       // TODO: Get type from parent blueprint, be able to select specialised type
```

Comment:
Perhaps I misunderstand, but this is fixed, right?
```suggestion
```

Reply:
Kind of, it now uses the passed type. Which should be fine